vulture/lines.py

```diff
@@ -23,27 +22,18 @@
     })
 
 
-# Reinventing wheel here to keep vulture lightweight.
-def memoize(func):
-    _cache = {}
+def get_last_line_number(node):
+    last_body = node.body[-1]
```

Comment:
The design is great, it even determines the last line for multi-line return statements and comments after return correctly (for python3). 😄